stdlib/LinearAlgebra/test/special.jl

```diff
@@ -434,4 +434,17 @@ end
     end
 end
 
+@testset "BiTriSym*Q' and Q'*BiTriSym" begin
+    dl = [1, 1, 1];
+    d = [1, 1, 1, 1];
+    Tri = Tridiagonal(dl, d, dl)
+    Bi = Bidiagonal(d, dl, :L)
+    Sym = SymTridiagonal(d, dl)
+    F = qr(ones(4, 1))
+    A=F.Q'
+    @test Tri*A ≈ (A*Tri)'
+    @test Bi*A ≈ (A'*Bi')'
+    @test Sym*A ≈ (A*Sym)'
```

Comment:
It might be better to make sure we use different code paths and test the other order of factors also, like
```suggestion
    A = F.Q'
    @test Tri*A ≈ Matrix(Tri)*A
    @test A*Tri ≈ A*Matrix(Tri)
    @test Bi*A ≈ Matrix(Bi)*A
    @test A*Bi ≈ A*Matrix(Bi)
    @test Sym*A ≈ Matrix(Sym)*A
    @test A*Sym ≈ A*Matrix(Sym)
```